src/main/java/frc/robot/SetupShuffleboard.java

```diff
@@ -119,15 +121,17 @@ public static void setupShuffleboard(SwerveSubsystem swerve, PizzaBoxSubsystem p
 
     Shuffleboard.getTab("Climb").add("climb prep", new PrepClimb(climbSubsystem, swerve, armSubsystem, reactionSubsystem)).withPosition(4, 0);
     Shuffleboard.getTab("Climb").add("climb & shoot", new ClimbAndShoot(climbSubsystem, swerve, armSubsystem, pizzaBoxSubsystem, reactionSubsystem)).withPosition(5, 0);
-    Shuffleboard.getTab("Climb").add("unclimb1", new UnClimb(climbSubsystem)).withPosition(4, 1);
-    Shuffleboard.getTab("Climb").add("unclimb2", new UnClimbPartTwoThatWillBringDownTheMotor(climbSubsystem, reactionSubsystem)).withPosition(5, 1);
+    Shuffleboard.getTab("Climb").add("unclimb1", new UnClimb(climbSubsystem, armSubsystem, reactionSubsystem)).withPosition(4, 1);
+    Shuffleboard.getTab("Climb").add("unclimb2", new UnClimbPartTwoThatWillBringDownTheMotor(climbSubsystem, swerve, armSubsystem, reactionSubsystem)).withPosition(5, 1);
 
     Shuffleboard.getTab("Climb").add("STOP CLIMB!!!!", new StopClimb(climbSubsystem)).withSize(2, 1).withPosition(2, 2);
 
     Shuffleboard.getTab("Climb").addDouble("Reaction Bar Angle", ()-> reactionSubsystem.getPos()).withPosition(9, 4);
 
 
     Shuffleboard.getTab("System Check").add("check", new SystemCheck(armSubsystem, climbSubsystem, intakeSubsystem, pizzaBoxSubsystem, reactionSubsystem, swerve));
+  
+    Shuffleboard.getTab("Arm").addDouble("distance", ()-> UtilMath.distanceFromBlueSpeaker(swerve.getPose()));
```

Comment:
Can we rename this to "distance from blue speaker"?

We should also upgrade the method to do distanceFromSpeaker and just check which speaker dynamically